modules/raft/src/main/java/org/apache/ignite/internal/raft/RaftGroupServiceImpl.java

```diff
@@ -566,9 +567,15 @@ private <R extends NetworkMessage> CompletableFuture<R> sendWithRetry(
      * @param stopTime Stop time.
      * @param fut The future.
      * @param <R> Response type.
+     * @param retriesCount Retries count.
```

Comment:
I'm not sure that this is a good name, from my point of view it could be interpreted like "how many retries will be done in the method `sendWithRetry". I would choose something like `retryCount` or `attemptCount`, but it is up to you. However I would definitely improve javadoc, just  `Retries count` is not enough.

Reply:
Renamed to retryCount. Javadoc impoved.